var/spack/repos/builtin/packages/lanl-bml/package.py

```diff
@@ -0,0 +1,48 @@
+##############################################################################
+# Copyright (c) 2017, Los Alamos National Security, LLC
+# Produced at the Los Alamos National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class LanlBml(CMakePackage):
+    """The basic matrix library (bml) is a collection of various matrix data
+    formats (in dense and sparse) and their associated algorithms for basic
+    matrix operations."""
+
+    homepage = "https://github.com/qmmd/bml"
+    url      = "https://github.com/qmmd/bml"
+
+    version('master', git='https://github.com/qmmd/bml', branch='master')
```

Comment:
Can you add at least one version that points to a release?  And call this version `develop`?  Spack prefers to check out the latest non-develop version, and we like that to be something fixed and verifiable.